From 198dbe381a4bc6a2c3af534df84a6e70119a5aa6 Mon Sep 17 00:00:00 2001
From: Zuohai Xu <zouhaix@marvell.com>
Date: Mon, 20 May 2019 16:41:49 +0800
Subject: [PATCH 249/386] arm64: dts: marvell: 9130-crb: Correct usb setting

The usb 2.0 should be enabled in config A of cn9130-crb.

Change-Id: I41d2f1abe2978dd5e460b00fa63ba2dbb2adf3b8
Signed-off-by: Zuohai Xu <zouhaix@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/9413
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb-A.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb-A.dts b/arch/arm64/boot/dts/marvell/cn9130-crb-A.dts
index c7a19386201d..a350584759db 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb-A.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb-A.dts
@@ -22,3 +22,15 @@
 		&cp0_comphy2 0
 		&cp0_comphy3 0>;
 };
+
+&cp0_usb3_0 {
+	status = "okay";
+	usb-phy = <&cp0_usb3_0_phy0>;
+	phy-names = "usb";
+};
+
+&cp0_usb3_1 {
+	status = "okay";
+	usb-phy = <&cp0_usb3_0_phy1>;
+	phy-names = "usb";
+};
-- 
2.17.1

